lib/newPage.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Title of the document</title>
+  </head>
+  <body>
+    The content of the document...
```

Comment:
I don't think you want any content in the `title` or `body`, you just want something that can be rendered in the preview without breaking.  I'd remove those two things, and this is probably good.